lib/symbolical_dynamics/euler_lagrange.py

```diff
@@ -184,14 +184,18 @@ def get_numerical_combined(self):
         self.h_num = lambdify([self.q, self.dq], self.h)
         return self.h_num
 
-    def get_headers(self, feature_names=None, dir=None):
+    def get_headers(self, feature_names=None, dir=None, create_cpp=False, file_name=None, class_name=None):
         """
         Create and save C headers
-        @param feature_names: custom names for headers, should be dict {'numerical_momentum": 'mom_name',
+        @param dict feature_names: custom names for headers, should be dict {'numerical_momentum": 'mom_name',
                                                                         'numerical_inertia': 'in_name' ... }
-        @param dir: directory to save headers, IT MUST END WITH '/' SYMBOL
+        @param string dir: directory to save headers, IT MUST END WITH '/' SYMBOL
+        @param bool create_cpp: create C++ class or not
+        @param string file_name: name for C++ file
+        @param string class_name: Name of the class for C++ file
         """
         # Default names for headers
+        global header_func
```

Comment:
There is no need to include "global" type of attribute here.

Reply:
Okay, I will fix it